### PR TITLE
CPA-20 AUTH refresh token

### PR DESCRIPTION
**Pull request by RudeAnna**
Created axiosInstance interceptor
NOTE still needs to be tested!

### CODE REVIEW

File: src/components/Modals/Modals.css [hunk 1]
```diff
@@ -0,0 +1 @@
+/* No CSS *//*# sourceMappingURL=Modals.css.map */
```
Comment (RudeMax) [inline src/components/Modals/Modals.css:1]: What is this comment about? No offence, just curious)

File: src/scss/commonCombainAll.css [hunk 2]
```diff
@@ -0,0 +1 @@
+/* No CSS *//*# sourceMappingURL=commonCombainAll.css.map */
```
Comment (RudeMax) [inline src/scss/commonCombainAll.css:1]: SAB

File: src/services/utils/axiosInstance.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import dayjs from "dayjs";
+import { BASE_URL } from "./endpointsConstuctor";
+
+export const AxiosInstance = axios.create();
+
+AxiosInstance.interceptors.request.use(async (req) => {
+	let authToken = localStorage.getItem("token");
+	if (!authToken) {
+		return req;
+	}
```
Comment (RudeMax) [inline src/services/utils/axiosInstance.js:12]: Shouldn't we name it more clear like `request` instead of `req`?

**Review by RudeMax — COMMENTED** [review]
Great and complicated job! Please see my comments.

File: src/services/utils/axiosInstance.js (hunk 3 above)
Comment (RudeAnna) [reply src/services/utils/axiosInstance.js:12]: "req" is common way to name a request object in middleware functions

**Review by MuscularSloth — APPROVED** [review]

**Review by yuriy1901 — APPROVED** [review]

**Pull request merged**